formats/need/publisher/details.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+    "required": [
+      "role",
+      "goal",
+      "benefit"
+    ],
+  "properties": {
+    "role": {
+      "type": "string"
```

Comment:
Thanks! For Needs, there are no descriptions, or fields that resemble descriptions, so there's no plan to add them later.